.golangci.yml

```diff
@@ -22,6 +22,11 @@ linters-settings:
       - octalLiteral
       - whyNoLint
       - wrapperFunc
+    settings:
+      rangeValCopy:
+        sizeThreshold: 512
```

Comment:
32 bytes is value taken from example on [https://golangci-lint.run/usage/linters/](https://golangci-lint.run/usage/linters/), so that's not related to this.

If we go down to 64 bytes, there would be approx. 15 more places requiring a change. 

There are many warnings with sizes 64, 72 and 80 bytes, so maybe we could consider 128 bytes, which would require only 5 more changes?

Reply:
I see, let's go with 128 bytes then